ironfish-rust/src/keys/mod.rs

```diff
@@ -197,12 +185,22 @@ impl SaplingKey {
     /// a seed. This isn't strictly necessary for private key, but view keys
     /// will need a direct mapping. The private key could still be generated
     /// using bip-32 and bip-39 if desired.
-    pub fn words_spending_key(&self, language: Language) -> Result<String, IronfishError> {
+    pub fn to_words(&self, language: Language) -> Result<String, IronfishError> {
         let mnemonic = Mnemonic::from_entropy(&self.spending_key, language)
             .map_err(|_| IronfishError::InvalidEntropy)?;
         Ok(mnemonic.phrase().to_string())
     }
 
+    /// takes words and turns them into spending key
```

Comment:
```suggestion
    /// Takes words and turns them into spending key
```

As a side note, it's not immediately clear what 'words' are, as someone new to this code. The doc-string here (and any other related functions) could be more descriptive about that, even something simple as "Takes a bip-39 phrase as a string..." would help